rbc_base/MotorParameters[3] parameters